src/index.js

```diff
@@ -69,6 +71,7 @@ export default class FloatAnchor extends React.Component<Props> {
     super(props);
     if (props.float) {
       this._portalEl = document.createElement('div');
+      this._portalEl.className = props.className || 'floating-portal';
```

Comment:
Let's not have a default classname.

Reply:
We also need to update the className when the prop changes.

Inside of componentWillReceiveProps, add an

```js
if (this._portalEl && newProps.floatContainerClassName !== this.props.floatContainerClassName) {
  this._portalEl.className = newProps.floatContainerClassName || '';
}
```